curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --answer--
 
-Placeholder answer
+It contains the "real" content of the element, such as text, images, or videos
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between `content-box` and `border-box` in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`content-box` includes padding and border, while `border-box` includes only content dimensions
 
 ---
 
-Placeholder distractor 2
+Both `content-box` and `border-box` include only the border dimensions
 
 ---
 
-Placeholder distractor 3
+`content-box` is used for 2D elements, while `border-box` is used for 3D elements
 
 #### --answer--
 
-Placeholder answer
+`content-box` includes only the content dimensions, while `border-box` includes content, padding, and border in its dimensions
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of CSS resets?
 
 #### --distractors--
 
-Placeholder distractor 1
+To apply default styles to all elements for faster loading
 
 ---
 
-Placeholder distractor 2
+To minimize the size of CSS files by removing unused styles
 
 ---
 
-Placeholder distractor 3
+To optimize images and fonts for better performance
 
 #### --answer--
 
-Placeholder answer
+To remove default browser styling and create a consistent baseline for styling across different browsers 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What are the common types of CSS resets?
 
 #### --distractors--
 
-Placeholder distractor 1
+Bootstrap Reset, Tailwind Reset, and Foundation Reset
 
 ---
 
-Placeholder distractor 2
+Simple Reset, Comprehensive Reset, and Adaptive Reset
 
 ---
 
-Placeholder distractor 3
+Minimal Reset, Advanced Reset, and Legacy Reset
 
 #### --answer--
 
-Placeholder answer
+Normalize.css, Eric Meyer's Reset, and HTML5 Reset
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `scale3d()` property value do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It rotates an element in 3D space   
 
 ---
 
-Placeholder distractor 2
+It skews an element on the x, y, and z axes
 
 ---
 
-Placeholder distractor 3
+It translates an element in 3D space
 
 #### --answer--
 
-Placeholder answer
+It scales an element along the x, y, and z axes
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How is the total width of an element calculated in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+Total width = width + margin + padding + border
 
 ---
 
-Placeholder distractor 2
+Total width = width + margin + padding
 
 ---
 
-Placeholder distractor 3
+Total width = width + margin + border
 
 #### --answer--
 
-Placeholder answer
+Total width = width + left padding + right padding + left border + right border
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the filter property in CSS do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It adjusts the visibility and opacity of an element
```

Comment:
A nitpick but by setting the opacity at 0%, you can turn an element invisible. And you can change the background color of an element with it too. And by increasing the contrast, it could seem smoother.